### PR TITLE
[Fix] User is not able to target hidden menus with TAB

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska

## Issue
Fliplet/fliplet-studio#6420

## Description
Active will change the display from none to block.

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

**Comment by tonytlwu** [comment]
@MaksymShokin Can you add a screencast for how the menu expands and collapses before and after this change? I believe the transforms were used to facilitate a smooth animation.

**Comment by MaksymShokin** [comment]
@tonytlwu 
Circle menu seems fine to me https://streamable.com/z4yajl

**Comment by tonytlwu** [comment]
@MaksymShokin You need to compare how this works in production. For example, you can see that there's a change in transition here https://share.getcloudapp.com/YEuow6BD

Please provide the same comparison for https://github.com/Fliplet/fliplet-menu-sequential/pull/18 and https://github.com/Fliplet/fliplet-menu-bottom-bar/pull/31

**Comment by MaksymShokin** [comment]
@tonytlwu 
I do not see any difference between your video and my video.

**Comment by tonytlwu** [comment]
Note that in my video when the menu opens, the menu items slide and fade in from the left.

**Comment by MaksymShokin** [comment]
Fixed https://streamable.com/w92bw3

File: css/menu.css [hunk 1]
```diff
@@ -129,7 +130,7 @@ body.disableScroll {
   padding-bottom: 30px;
   padding-bottom: calc(env(safe-area-inset-bottom) + 30px);
   z-index: 9;
-  -webkit-transform: translate3d(0, -100%, 0);
+  -webkit-transform: translate3d(0, -100%, 0);
```
Comment (tonytlwu) [inline css/menu.css]:
```suggestion
  -webkit-transform: translate3d(0, -100%, 0);
```

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: css/menu.css (hunk 1 above)
Comment (MaksymShokin) [reply css/menu.css]: Fixed

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Low
Affected: Circle app menu
Verify: Menu works as usual

**Pull request merged**